infra/site-cookbooks/cassandra-linux-tweaks/recipes/default.rb

```diff
@@ -7,61 +7,22 @@
 # All rights reserved - Do Not Redistribute
 #
 
-
-bash 'update net.core.rmem_max' do
-  code <<-EOH
-    sysctl net.core.rmem_max=16777216
-  EOH
-end
-
-bash 'update net.core.wmem_max' do
-  code <<-EOH
-    sysctl net.core.wmem_max=16777216
-  EOH
-end
-
-bash 'update net.core.rmem_default' do
-  code <<-EOH
-    sysctl net.core.rmem_default=16777216
-  EOH
-end
-
-bash 'update net.core.wmem_default' do
-  code <<-EOH
-    sysctl net.core.wmem_default=16777216
-  EOH
-end
-
-bash 'update net.core.optmem_max' do
-  code <<-EOH
-    sysctl net.core.optmem_max=40960    
-  EOH
-end
-
-bash 'update net.ipv4.tcp_rmem' do
-  code <<-EOH
-    sysctl net.ipv4.tcp_rmem="4096 87380 16777216"
-  EOH
-end
-
-bash 'update net.ipv4.tcp_wmem' do
-  code <<-EOH
-    sysctl net.ipv4.tcp_wmem="4096 65536 16777216"
-  EOH
+file "/home/cassandra/startup-script-cassandra-tweaks.sh" do
+  source "startup-script-cassandra-tweaks.sh"
 end
 
 template '/etc/security/limits.d/cassandra.conf' do 
   source 'security-cassandra.conf'
 end
 
-bash 'zone_reclaim_mode' do
+bash "chown cassandra.cassandra startup-script-cassandra-tweaks.sh" do
   code <<-EOH
-    echo 0 > /proc/sys/vm/zone_reclaim_mode
+    chown cassandra.cassandra /home/cassandra/startup-script-cassandra-tweaks.sh
   EOH
 end
 
-bash "disable swap" do
+bash 'execute startup-script-cassandra-linux-tweaks.sh' do
   code <<-EOH
-    swapoff --all
+    ./startup-script-cassandra-linux-tweaks.sh
```

Comment:
we just execute the "manually" (while cheffing) so the "now cheffed" instance is the exact same thing as it will be after reboot when these scripts trigger